.github/workflows/UnitTests-core.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           tar -zcvf ${{ inputs.artifact_suffix }}-jdk${{ matrix.jdk }}-logs.tar.gz ${{ inputs.artifact_suffix }}-artifacts
       - name: Publish Test Report
         env:
-          NODE_OPTIONS: "--max_old_space_size=4096"
+          NODE_OPTIONS: "--max_old_space_size=8192"
```

Comment:
Node? So it's a nodejs thing? I have to update the "other languages: section of my recent docs update then 😅

Reply:
yup, these reporters uses node.js. GH can infer that no? currently it shows

Java 99.6%
Other 0.4%